erlang/potatoserver/test/my_crypto_test.erl

```diff
@@ -1,6 +1,6 @@
 -module(my_crypto_test).
 
--export([my_crypto_test/0]).
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
oh I didn't realize we needed to do this ʘ‿ʘ
does that mean none of the unit tests were running before?

Reply:
we were good before!

this just auto exports all functions ending with "_test"

Reply:
such actually reading through eunit documentation